src/components/structural/MaterialType.js

```diff
@@ -0,0 +1,7 @@
+export const MaterialType = {
```

Comment:
seeing as this isn't a component, it should probably be placed in "src/constants/MaterialType.js"